wirecard/oxid-ee
# WDPD-42: Giropay payment method

### This PR

* Adds Giropay payment method

### Notes

* No backend operations are available for Giropay at this time (see https://github.com/wirecard/magento2-ee/wiki/giropay)

### How to Test

* Open the admin, go to _Extensions → Modules_ and activate the module
* Go to _Shop Settings → Payment Methods_ and activate _Wirecard Giropay_
* Open the shop and make a purchase using Giropay as a payment method
* For the BIC, use a BIC of [one of the supporting banks](https://www.giropay.de/kaeufer/) (e.g. `BYLADEM1001`, `BYLADEM1KIS`)
* Abort the payment and use the [Test Bank Account BIC](https://document-center.wirecard.com/display/PTD/giropay#giropay-TestCredentials)
* In the test account screen, trigger a successful payment by entering these values:
  * sc: `10`
  * extensionSc: `4000`
* Confirm that the payment was successful by going to Wirecard _Transactions → Transactions_ and _Administer Orders → Orders_ in the admin

### Jira Links

* WDPD-42: https://jira.parkside.at/browse/WDPD-42
